myokit/_sim/fiber_tissue.py

```diff
@@ -365,17 +365,20 @@ def _create_backend(self):
 
         # Define libraries
         libs = []
+        flags = []
```

Comment:
I think I've managed to fix the OpenCL issues on os/x, but in a separate PR. We'll have to find some way to revert the changes to these three files before we can merge this PR

Reply:
reverted and merged the develop branch, bit this didn't fix the bug, see comment below.